platform/linux-generic/odp_queue_scalable.c

```diff
@@ -333,12 +333,12 @@ static odp_schedule_group_t queue_sched_group(odp_queue_t handle)
 	return qentry_from_int(queue_from_ext(handle))->s.param.sched.group;
 }
 
-static int queue_lock_count(odp_queue_t handle)
+static uint32_t queue_lock_count(odp_queue_t handle)
 {
 	queue_entry_t *queue = qentry_from_int(queue_from_ext(handle));
 
 	return queue->s.param.sched.sync == ODP_SCHED_SYNC_ORDERED ?
-		(int)queue->s.param.sched.lock_count : -1;
+		(int)queue->s.param.sched.lock_count : 0;
```

Comment:
Remove cast (int)

Reply:
This parameter is uint32_t hence I moved from error from -1 to 0.

Reply:
That's OK, but now cast to "int" should not be done anymore. The function returns now uint32_t and not int.

static uint32_t queue_lock_count(odp_queue_t handle)